htdocs/comm/propal/card.php

```diff
@@ -1793,6 +1793,7 @@
 	$deposit_percent = GETPOST('cond_reglement_id_deposit_percent', 'alpha');
 	$mode_reglement_id = GETPOSTINT('mode_reglement_id');
 	$fk_account = GETPOSTINT('fk_account');
+	$datepropal = (empty($datepropal) ? (!getDolGlobalString('MAIN_AUTOFILL_DATE') ? -1 : '') : $datepropal);
```

Comment:
MAIN_AUTOFILL_DATE was introduced for "payment".
You can use MAIN_AUTOFILL_DATE_PROPOSAL